assets/src/js/nationality.js

```diff
@@ -0,0 +1,276 @@
+    "Egyptian",
+    "Emirati",
+    "English",
+    "Equatorial Guinean",
+    "Eritrean",
+    "Estonian",
+    "Ethiopian",
+    "Faroese",
+    "Fijian",
+    "Filipino",
+    "Finnish",
+    "French",
+    "Gabonese",
+    "Gambian",
+    "Georgian",
+    "German",
+    "Ghanaian",
+    "Gibraltarian",
+    "Greek",
+    "Greenlandic",
+    "Grenadian",
+    "Guamanian",
+    "Guatemalan",
+    "Citizen of Guinea-Bissau,",
+    "Guinean",
+    "Guyanese",
+    "Haitian",
+    "Honduran",
+    "Hong Konger",
+    "Hungarian",
+    "Icelandic",
+    "Indian",
+    "Indonesian",
+    "Iranian",
+    "Iraqi",
+    "Irish",
+    "Israeli",
+    "Italian",
+    "Ivorian",
+    "Jamaican",
+    "Japanese",
+    "Jordanian",
+    "Kazakh",
+    "Kenyan",
+    "Kittitian",
+    "Citizen of Kiribati",
+    "Kosovan",
+    "Kuwaiti",
+    "Kyrgyz",
+    "Lao",
+    "Latvian",
+    "Lebanese",
+    "Liberian",
+    "Libyan",
+    "Liechtenstein citizen",
+    "Lithuanian",
+    "Luxembourger",
+    "Macanese",
+    "Macedonian",
+    "Malagasy",
+    "Malawian",
+    "Malaysian",
+    "Maldivian",
+    "Malian",
+    "Maltese",
+    "Marshallese",
+    "Martiniquais",
+    "Mauritanian",
+    "Mauritian",
+    "Mexican",
+    "Micronesian",
+    "Moldovan",
+    "Monegasque",
+    "Mongolian",
+    "Montenegrin",
+    "Montserratian",
+    "Moroccan",
+    "Mosotho",
+    "Mozambican",
+    "Namibian",
+    "Nauruan",
+    "Nepalese",
+    "New Zealander",
+    "Nicaraguan",
+    "Nigerian",
+    "Nigerien",
+    "Niuean",
+    "North Korean",
+    "Northern Irish",
+    "Norwegian",
+    "Omani",
+    "Pakistani",
+    "Palauan",
+    "Palestinian",
+    "Panamanian",
+    "Papua New Guinean",
+    "Paraguayan",
+    "Peruvian",
+    "Pitcairn Islander",
+    "Polish",
+    "Portuguese",
+    "Prydeinig",
+    "Puerto Rican",
+    "Qatari",
+    "Romanian",
+    "Russian",
+    "Rwandan",
+    "Salvadorean",
+    "Sammarinese",
+    "Samoan",
+    "Sao Tomean",
+    "Saudi Arabian",
+    "Scottish",
+    "Senegalese",
+    "Serbian",
+    "Citizen of Seychelles",
+    "Sierra Leonean",
+    "Singaporean",
+    "Slovak",
+    "Slovenian",
+    "Solomon Islander",
+    "Somali",
+    "South African",
+    "South Korean",
+    "South Sudanese",
+    "Spanish",
+    "Sri Lankan",
+    "St Helenian",
+    "St Lucian",
+    "Stateless",
+    "Sudanese",
+    "Surinamese",
+    "Swazi",
+    "Swedish",
+    "Swiss",
+    "Syrian",
+    "Taiwanese",
+    "Tajik",
+    "Tanzanian",
+    "Thai",
+    "Togolese",
+    "Tongan",
+    "Trinidadian",
+    "Tristanian",
+    "Tunisian",
+    "Turkish",
+    "Turkmen",
+    "Turks and Caicos Islander",
+    "Tuvaluan",
+    "Ugandan",
+    "Ukrainian",
+    "Uruguayan",
+    "Uzbek",
+    "Vatican citizen",
+    "Citizen of Vanuatu",
+    "Venezuelan",
+    "Vietnamese",
+    "Vincentian",
+    "Wallisian",
+    "Welsh",
+    "Yemeni",
+    "Zambian",
+    "Zimbabwean"
+];
+var autocompleteConfig1 = {
```

Comment:
can we change the var/cost name to use just alphabets with camelCasing. Eg: autoCompleteConfigOne. Same for other instances.